addons/api/addon/generated/models/host-catalog.js

```diff
@@ -83,13 +86,17 @@ export default class GeneratedHostCatalogModel extends BaseModel {
   // AWS static credentials
   @attr('string', {
     for: 'plugin',
+    credentialType: 'static-credential',
```

Comment:
Asking for learning puposes: what is the goal of adding `credentialType` within the model? I think we had before types without the need of adding this field, so I wanted to understand the reasons.

Reply:
AWS plugin supports two credential types- static and dynamic, the fields were listed in the serializer, but now I moved it up to the models, so we don't list out the fields in the serializer and can make use of the `for`